nixos/modules/services/misc/klipper.nix

```diff
@@ -125,6 +125,12 @@ in
                 type = path;
                 description = lib.mdDoc "Path to firmware config which is generated using `klipper-genconf`";
               };
+              package = mkOption {
+                type = types.package;
+                default = pkgs.klipper-firmware;
+                defaultText = lib.literalExpression "pkgs.klipper-firmware";
+                description = lib.mdDoc "Firmware package to build for this printer.";
+              };
```

Comment:
```suggestion
              package = lib.mkPackageOptionMD pkgs "klipper-firmware" { };
```

Reply:
Is there a reason to remove descriptions?

Also, if going this route, why `mkPackageOptionMD` (which is just an alias for `mkPackageOption`), and not `mkPackageOption` directly?

Reply:
Soon, there's no reason to use it anymore, but it has been the common way to do things during 23.05 in case something was going to be backported. `mkPackageOption` does not work in 23.05, in the same way that dropping `mdDoc` does not work, but that's about to change.

If you feel extra description is needed, you can set the `extraDescription` flag inside the last `{ }`.